receiver/receivercreator/README.md

```diff
@@ -439,3 +439,296 @@ service:
 
 The full list of settings exposed for this receiver are documented [here](./config.go)
 with detailed sample configurations [here](./testdata/config.yaml).
+
+
+## Generate receiver configurations from provided Hints
+
+Note: When hints feature is enabled if hints are present for an endpoint no receiver templates will be evaluated.
+
+Currently this feature is only supported for K8s environments and the `k8sobserver`.
+
+The metrics' hints feature for K8s is enabled with the following setting:
+
+```yaml
+receiver_creator/metrics:
+  watch_observers: [ k8s_observer ]
+  hints:
+    k8s:
+      metrics:
+       enabled: true
+```
+
+The logs' hints feature for K8s is enabled with the following setting:
+
+```yaml
+receiver_creator/logs:
+  watch_observers: [ k8s_observer ]
+  hints:
+    k8s:
+      logs:
+       enabled: true
+        # indicate whether only logs from annotated Pods are collected or from all by default
+       # default: true|false 
+```
+
+Find bellow the supported annotations that user can define to automatically enable receivers to start collecting metrics/logs signals from the target Pods/containers.
+
+### Supported metrics annotations
+
+#### Enable/disable discovery
+
+`io.opentelemetry.discovery.metrics/enabled` (example: `"true"`)
+
+By default `"true"`.
+
+#### Define scraper
+
+`io.opentelemetry.discovery.metrics/scraper` (example: `"nginx"`)
```

Comment:
Are scrapers names with ID supported? E.g. `nginx/server`?

Although it would be great to support this, we can keep it for future improvements.

Reply:
We could figure out how to support this in the future, yeap.

At the moment the ID is constructed combining the provided name, the Pod's UID and the port. This would ensure the uniqueness of the ID for the Collector instance I think.

Do you have any specific example of how `nginx/server` would be used by a user?